flink-python/pyflink/table/tests/test_file_system_descriptor.py

```diff
@@ -0,0 +1,33 @@
+################################################################################
```

Comment:
Merge test_descriptor_end_to_end.py, test_file_system_descriptor.py and other descriptor tests into `test_descriptor.py`

Reply:
Done.